modules/javafx.graphics/src/main/java/com/sun/glass/ui/win/WinApplication.java

```diff
@@ -376,7 +376,14 @@ public String getDataDirectory() {
     }
 
     @Override
-    protected native int _getKeyCodeForChar(char c);
+    protected int _getKeyCodeForChar(char c) {
+        // This platform has migrated to getKeyCanGenerateCharacter
+        // so getKeyCodeForChar will no longer be called.
+        return 0;
```

Comment:
If you are sure it is never called, then how about going all in and throwing an exception here? Currently it just returns an incorrect result.